public/docs/advanced/commands.md

```diff
@@ -132,55 +218,94 @@ Optional parameters:
 
 * `--clean`: Remove all network artifacts that aren't associated with a named network.
 
+
+### opcode
+
+Print the compiled opcodes for a given contract.
+
+```shell
+truffle opcode <contract_name>
+```
+
+Parameters:
+
+* `<contract_name>`: Name of the contract to print opcodes for. Must be a contract name, not a file name.
+
+
 ### publish
 
 Publish a package to the Ethereum Package Registry.
 
-```none
-$ truffle publish
+```shell
+truffle publish
 ```
 
-All parameters are pulled from your project's configuration file. See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
+Takes no arguments. All parameters are pulled from your project's configuration file. See the [Package Management with EthPM](/docs/getting_started/packages-ethpm) section for more details.
+
 
 ### serve
 
-Serve the built app from `http://127.0.0.1:8080`, rebuilding and redeploying changes as needed. Like `truffle watch`, but with the web server component added.
+Serve the built app from `http://127.0.0.1:8080`, rebuilding and redeploying changes as needed. Similar to `truffle watch`, but with the web server component added.
 
-```none
-$ truffle serve
+```shell
+truffle serve [-p <port>] [--network <name>]
 ```
 
 Optional parameters:
 
-* `-p port`: Specify the port to serve on. Default is 8080.
-* `--network name`: Specify the network to use, using artifacts specific to that network.
+* `-p <port>`: Specify the port to serve on. Default is 8080.
+* `--network <name>`: Specify the network to use, using artifacts specific to that network. Network name must exist in the configuration.
+
 
 ### test
 
-Run all tests within the `./test` directory, or optionally run a single test.
+Run Mocha and Solidity tests.
 
-```none
-$ truffle test [/path/to/test/file]
+```shell
+truffle test <test_file> [--compile-all] [--network <name>] [--verbose-rpc]
 ```
 
+Will run some or all tests within the `test/` directory, as specified.
+
+Parameters:
+
+* `<test_file>`: Name of the test file to be run. Can include path information if the file does not exist in the current directory.
+
 Optional parameters:
 
-* `--compile-all`: Compile all contracts instead of intelligently choosing.
+* `--compile-all`: Compile all contracts instead of intelligently choosing which contracts need to be compiled.
+* `--network <name>`: Specify the network to use, using artifacts specific to that network. Network name must exist in the configuration.
 * `--verbose-rpc`: Log communication between Truffle and the RPC.
-* `--network name`: Specify the network to use, using artifacts specific to that network.
+
+### unbox
+
+Unbox Truffle project.
```

Comment:
Probably need to fill this sentence out. `Unbox a pre-built Truffle project.`

Reply:
Will edit the description in `truffle-core`.

So as to avoid using the word in its definition, I think "Downloads a Truffle Box, a pre-built Truffle project" might be more appropriate.